docs/prover/checking/coverage-info.md

```diff
@@ -0,0 +1,147 @@
+
+The visualization here consists only of 2 colors: `green` means that the command is needed and `red` means that the command is not needed (i.e. not in the unsat core). In particular, Figures 4a and 4b show the TAC visualization for the Tautology example. 
+
+![](tautology-tac-a-new.png)
+Figure 4a: The visualization of the Tautology example on TAC, part A. 
+
+![](tautology-tac-b-new.png)
+Figure 4b: The visualization of the Tautology example on TAC, part B.
+
+Namely, only the following TAC commands are in the unsat core:
+
+```
+→ require other != manager && other != e.msg.sender
+I95 = other287 SPEC
+I96 = manager269 SPEC
+tmp292293 = I95==I96 SPEC
+B97 = !tmp292293 SPEC
+...
+certoraAssume288289 = B97&&B102 SPEC
+assume certoraAssume288289
+...
+→ assert newManager != other || newManager != manager
+I158 = newManager419 SPEC
+I159 = other287 SPEC
+tmp441442 = I158==I159 SPEC
+B160 = !tmp441442 SPEC
+I161 = newManager419 SPEC
+I162 = manager269 SPEC
+tmp449450 = I161==I162 SPEC
+B163 = !tmp449450 SPEC
+certoraAssert437438 = B160||B163 SPEC
+assert certoraAssert437438,
+```
+
+Also, notice that some commands in the TAC dump are followed by "SPEC" or "SOL" with a violet background color; these are commands for which we have a mapping to `.spec` and `.sol` files, respectively. 
+
+## Basic vs. advanced mode
+
+The `--coverage_info` flag takes three possible values: `none`, `basic` and `advanced`:
+1. `none` means no unsat core analysis and no visualization, 
+2. `basic` means relatively fast bust possibly very imprecise analysis, 
+3. and `advanced` means possibly slow but more precise analysis. 
+
+In particular, the Certora Prover internally does many various transformations of the TAC program before it converts the TAC program to SMT. These transformations usually simplify the TAC program and make the whole verification task easier and faster. However, some of these transformations might lead to loosing a mapping between the initial and the final TAC program, and consequently also loosing a mapping between the SMT encoding and the input `.sol` and `.spec` files. In such case, some lines/commands in the `.spec` and `.sol` visualization might be marked as not needed even if they are actually needed; the other direction, i.e. spurious red lines, is also possible. The `basic` mode keeps all the TAC transformations and hence is faster but possibly more imprecise, whereas the `advanced` mode turns off some of the transformations to make the analysis more precise. 
+
+## Multiple unsat cores
+In general, the SMT formula can have multiple unsat cores; in fact, even exponentially many w.r.t. the number of assertions in the formula. By default, we generate just a single unsat core per rule/method/invariant. Use can use the flag `--prover_args "-numOfUnsatCores <k>"` to instruct the Certora Prover to generate up `k` unsat cores for each rule/method/invariant. At this moment, we support `k > 1` only for the TAC visualizations. In particular, 
```

Comment:
are you talking about "minimal unsat cores" here? -- is this implicit by your definition?..
(otherwise it's even less suprising that there are many, naturally..)

Reply:
My definition says "We call a minimal unsatisfiable subset of `A` an *unsat core* of `A`.". There is no *minimal* unsat core. Just *unsat core*.

Reply:
I removed this section